src/api/app/models/scm_webhook.rb

```diff
@@ -44,6 +44,14 @@ def pull_request_event?
     github_pull_request? || gitlab_merge_request?
   end
 
+  def allowed_pull_request_or_merge_request_event?
+    if github_pull_request?
+      ScmWebhookEventValidator::ALLOWED_PULL_REQUEST_ACTIONS.include?(@payload[:action])
+    else
+      ScmWebhookEventValidator::ALLOWED_MERGE_REQUEST_ACTIONS.include?(@payload[:action])
```

Comment:
I think we also need to filter out other events like editing a PR description, that now are reaching our code but don't have any workflow defined.

Reply:
It's updated.